src/Core/Models/Response/IdentityTokenResponse.cs

```diff
@@ -21,5 +21,6 @@ public class IdentityTokenResponse
         public KdfType Kdf { get; set; }
         public int? KdfIterations { get; set; }
         public bool ForcePasswordReset { get; set; }
+        public string KeyConnectorUrl { get; set; } = null;
```

Comment:
I don't think you need the default assignment here, see my other comment in our thread below

Reply:
removed null assignment from strings
68267be